src/panoptes/pocs/focuser/astromechanics.py

```diff
@@ -22,14 +22,17 @@ class Focuser(AbstractSerialFocuser):
     as they are marked with the decorator @abstractmethod, we have to override them.
     """
 
-    def __init__(self, name='Astromechanics Focuser', model='Canon EF-232', vendor_id=None,
-                 product_id=None, *args, **kwargs):
+    def __init__(self, name='Astromechanics Focuser', model='Canon EF-232', port=None,
```

Comment:
If I make a focuser instance without any args, there is an error while raising the `Exception`:

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-4-85c5d668a05e> in <module>
----> 1 f = Focuser()

<ipython-input-3-686809ae444a> in __init__(self, name, model, port, vendor_id, product_id, *args, **kwargs)
     19         # Check if have device, raise error.NotFound if unable to find.
     20         try:
---> 21             port = find_serial_port(vendor_id, product_id)
     22             self._vendor_id = vendor_id
     23             self._product_id = product_id

/opt/conda/lib/python3.9/site-packages/panoptes/utils/serial/device.py in find_serial_port(vendor_id, product_id, return_all)
    114     else:
    115         raise error.NotFound(
--> 116             f'No serial ports for vendor_id={vendor_id:x} and product_id={product_id:x}')
    117
    118

TypeError: unsupported format string passed to NoneType.__format__
```